src/models/BibDetails.ts

```diff
@@ -124,17 +140,10 @@ export default class BibDetails {
           detail = this.buildInternalLinkedDetail(fieldMapping)
         else if (fieldMapping.field === "subjectLiteral")
           detail = this.subjectHeadings
-        else if (fieldMapping.field === "extent") detail = this.extent
-        else if (fieldMapping.field === "owner")
-          detail = this.bib.owner && {
```

Comment:
Where do the owner and extent attributes get called now that this has been removed?

Reply:
Let me know if you want me to explain! It's a little tricky but I'm hoping this streamlines things a little.